pkg/idutil/imagewalker/imagewalker.go

```diff
@@ -22,6 +22,7 @@ import (
 	"regexp"
 	"strings"
 
+	"github.com/distribution/reference"
```

Comment:
Same as above.
Unless there is a reason to use `distribution/reference`, one should use nerdctl referenceutils instead.